vim/delete_comments.md

```diff
@@ -0,0 +1,9 @@
+# Delete all comments in one go
+
+Every Rails app you generate comes with many files such as routes that includes lots of comments.
+
+You can delete all these comments at once using the following vim command:
+
+```vimscript
+:g:^#:normal dd
```

Comment:
It could be even shorter:

```vimscript
:g/^#/d
```

For every line matching `^#`, run `d` (delete).